tap_github/streams.py

```diff
@@ -412,3 +401,101 @@ def post_process(self, row: dict, context: Optional[dict] = None) -> dict:
             ),
         ),
     ).to_dict()
+
+
+class CommitsStream(GitHubStream):
```

Comment:
@aaronsteers now that we start to have a lot more streams in this file, what is your preferred strategy for ordering them? Alphabetical order? Splitting into files? Eg. one file for issues and issues-related streams?

Reply:
@ericboucher - Good question. I'm not sure. I think alpha-sorting makes sense to some degree, except that we would want parent streams (repositories, for instance) so they can easily be referenced by children. One file per stream seems a bit messy but might actually be cleaner - perhaps under a `streams` subfolder? Hard to say. 

What do you think?

Reply:
I think one file per "parent" stream would make sense ultimately, potentially with a few shared utils for similar override functions.

Reply:
@ericboucher - That works for me. I do also think moving into a streams subfolder would be helpful, and if so, the `streams/__init__.py` file can import all types so the tap could simply import all types from `streams` without having to reference each file.

Reply:
@ericboucher - Happy to take a re-org as a separate PR if you'd like. I'll go ahead and merge this in the meanwhile. Thanks again.